test/Conversion/ArcToLLVM/lower-arc-to-llvm.mlir

```diff
@@ -222,3 +222,56 @@ func.func @WriteArray(%arg0: !arc.state<!hw.array<4xi1>>, %arg1: !hw.array<4xi1>
   arc.state_write %arg0 = %arg1 : <!hw.array<4xi1>>
   return
 }
+
+func.func @assert_with_cond() {
+  // CHECK-LABEL: @assert_with_cond(
+  // CHECK-NEXT:    [[COND:%.+]] = llvm.mlir.constant(false) : i1
+  // CHECK-NEXT:    llvm.cond_br [[COND]], ^[[TRUE_BRANCH:[[:alnum:]]+]], ^[[FALSE_BRANCH:[[:alnum:]]+]]
```

Comment:
Never seen `[[:alnum:]]` before. What does it do? Is it the same as `[0-9]`?

Reply:
It means alpha-numeric. May be I am missing something but you will find it a lot in mlir tests https://github.com/llvm/llvm-project/blob/main/mlir/test/Dialect/Arith/canonicalize.mlir#L2306

Reply:
Ah, that's neat. Thanks!